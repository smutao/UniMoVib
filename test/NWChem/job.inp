a test job

 $contrl
   qcprog="nwchem"
 $end

 $qcdata
   fchk="h2o.out"
   hess="dft.hess"
   ddip="dft.fd_ddipole"
 $end
