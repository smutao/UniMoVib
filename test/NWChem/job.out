 =======================================================================================================================
       _/     _/                _/     _/            _/     _/
      _/     _/            _/  _/|  _/_/            _/     _/  _/  _/        UniMoVib:          A unified interface for
     _/     _/   _/_/_/       _/ |_/ _/   _/_/_/   _/     _/      _/           molecular harmonic vibrational frequency
    _/     _/  _/    _/  _/  _/  _/ _/  _/    _/  _/     _/  _/  _/_/_/        calculations.
   _/     _/  _/    _/  _/  _/     _/  _/    _/  _/    _/   _/  _/   _/
  _/     _/  _/    _/  _/  _/     _/  _/    _/   _/  _/    _/  _/   _/       Ver. 1.0.0, MAY 02, 2017
   _/_/_/   _/    _/  _/  _/     _/   _/_/_/     _/_/     _/  _/_/_/         https://github.com/zorkzou/UniMoVib
 =======================================================================================================================


 QC Program:      NWCHEM
 FCHK file:       h2o.out
 HESS file:       dft.hess
 DDIP file:       dft.fd_ddipole


 Cartesian coordinates (Angstrom)
 ------------------------------------------------------------------------------------------
   No.   Atom    ZA                 X             Y             Z                Mass
 ------------------------------------------------------------------------------------------
     1    O       8            0.12132256    0.00000000    0.00000000           15.99491000
     2    H       1           -0.48529023    0.78370622    0.00000000            1.00782500
     3    H       1           -0.48529023   -0.78370622    0.00000000            1.00782500
 ------------------------------------------------------------------------------------------


 <<< SYMMETRY >>>
 Molecular Point Group (Z)   :   C2v 
 Molecular Point Group (Z+M) :   C2v 


 ************************************
 ***  Properties of Normal Modes  ***
 ************************************

 Results of vibrations:
 Normal frequencies (cm**-1), reduced masses (AMU),
 force constants (mDyn/A), IR intensities (km/mol)

                                                   1                                 2                                 3
          Irreps                                  A1                                A1                                B2
     Frequencies                           1698.4921                         3498.8103                         3643.7539
  Reduced masses                              1.0896                            1.0388                            1.0819
 Force constants                              1.8520                            7.4924                            8.4630
  IR intensities                             44.5556                            8.0693                            0.0076
        Atom  ZA               X         Y         Z             X         Y         Z             X         Y         Z
           1   8         0.07387  -0.00000   0.00000       0.04545   0.00000   0.00000      -0.00000  -0.07029   0.00000
           2   1        -0.58618  -0.39200   0.00000      -0.36070   0.60734   0.00000      -0.43174   0.55779   0.00000
           3   1        -0.58618   0.39200   0.00000      -0.36070  -0.60734   0.00000       0.43174   0.55779   0.00000

 Results of translations and rotations:

                                                   1                                 2                                 3
          Irreps                                  A1                                B2                                B1
     Frequencies                              1.0077                            1.7832                           -1.8877
  Reduced masses                              6.0035                            6.0035                            6.0035
 Force constants                              0.0000                            0.0000                           -0.0000
  IR intensities                              0.0000                            0.0000                            0.0000
        Atom  ZA               X         Y         Z             X         Y         Z             X         Y         Z
           1   8         0.57735   0.00000   0.00000       0.00000   0.57735   0.00000       0.00000   0.00000   0.57735
           2   1         0.57735   0.00000   0.00000       0.00000   0.57735   0.00000       0.00000   0.00000   0.57735
           3   1         0.57735   0.00000   0.00000       0.00000   0.57735   0.00000       0.00000   0.00000   0.57735

                                                   4                                 5                                 6
          Irreps                                  B1                                A2                                B2
     Frequencies                             17.1599                          -11.3349                            4.1972
  Reduced masses                              1.1259                            1.0078                            1.0459
 Force constants                              0.0002                           -0.0001                            0.0000
  IR intensities                            339.2548                            0.0000                          117.8197
        Atom  ZA               X         Y         Z             X         Y         Z             X         Y         Z
           1   8         0.00000   0.00000   0.08876       0.00000   0.00000  -0.00000       0.00000  -0.05041   0.00000
           2   1         0.00000   0.00000  -0.70432       0.00000   0.00000  -0.70711       0.58197   0.40005   0.00000
           3   1         0.00000   0.00000  -0.70432       0.00000   0.00000   0.70711      -0.58197   0.40005   0.00000


 *********************************************
 ***   Thermal Contributions to Energies   ***
 *********************************************

 Temperature               :       298.150000    Kelvin
 Pressure                  :         1.000000    Atm
 Molecular mass            :        18.010560    AMU
 Elec. total energy        :         0.000000    Hartree
 Scale factor of Frequency :         1.000000
 Rot. symmetry number      :         2
 The C2v  point group is used to calculate rotational entropy.

 Principal axes and moments of inertia in atomic units:
                                     1                   2                   3
     Eigenvalues --                  2.352290            4.420993            6.773283
           X                         0.000000            1.000000            0.000000
           Y                         1.000000            0.000000            0.000000
           Z                         0.000000            0.000000            1.000000

 Rotational temperatures            36.821056           19.591486           12.787568    Kelvin
 Rot. constants A, B, C             25.591946           13.616781            8.887815    cm**-1
                                   767.227245          408.220818          266.449998    GHz

 Thermal correction energies                              Hartree            kcal/mol
 Zero-point Energy                          :            0.020141           12.638923
 Thermal correction to Energy               :            0.022976           14.417719
 Thermal correction to Enthalpy             :            0.023920           15.010205
 Thermal correction to Gibbs Free Energy    :            0.002413            1.514040


 Job terminated correctly! Tue May 02 18:20:09 2017
